modules/tribes/client/components/helpers/getTribeBackground.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Generate background color + image styles for tribe panel
+ *
+ * usage in React: <div style={getTribeBackground(tribe, { quality: 'normal', dimensions: '742x496', isProgressive: true })}>...</div>
+ *
+ * @param {object} tribe - the tribe data
+ *
+ * @param {string} [dimensions=1024x768] - Set background image dimensions (width x height)
+ *   See https://uploadcare.com/documentation/cdn/#operation-scale-crop
+ *
+ * @param {string} [quality=lighter] - Set background image quality
+ *   Options: normal, better, best, lighter (default), lightest
+ *   See https://uploadcare.com/documentation/cdn/#operation-quality
+ *
+ * @param {boolean} [isProgressive=false] - Set progressive image loading
+ *   See https://uploadcare.com/documentation/cdn/#operation-progressive
+ *
+ * @returns {object}
+ */
+export default function getTribeBackground(tribe, { quality='lighter', dimensions='1024x768', isProgressive=false }={}) {
```

Comment:
minor point: `getTribeBackgroundStyle` would be slightly clearer to me, it wasn't immediately obvious that it was going to return me a style object I can use in a `style` attribute.

Reply:
Renamed in 2c37f3f38c0504bcd17b6f27000296e0d66fe448